screwdriver/testdata/screwdriver.yaml

```diff
@@ -0,0 +1,9 @@
+jobs:
+    main:
+        image: alpine
+        steps:
+            - install: echo install
+            - publish: echo publish
+        environment:
+            test_env: hoge
```

Comment:
this yaml causes `ValidationError`.

> echo "ValidationError: child "jobs" fails because [child "main" fails because [child "environment" fails because ["test_env" only supports uppercase letters, digits, and underscore (cannot start with digit)]]]"; exit 1

Reply:
fixed